editor/src/components/editor/store/dispatch.tsx

```diff
@@ -471,16 +475,45 @@ export function editorDispatch(
   }
 
   const isLoaded = frozenEditorState.isLoaded
+  /**
+   * We only want to trigger a save for code result cache change if the project was already saved.
+   * This is to prevent accidentally forking projects before the user even edits anything in them.
+   */
+  const saveBecauseCodeResultCacheChanged = updateCodeResultCache && frozenEditorState.alreadySaved
```

Comment:
Should we even still trigger a save on a change to the code result cache? I have a feeling this is related to updating the preview rather than actually storing the result of the cache change

Reply:
the original change was a fix so that if you introduce a compile error and reload, the error persists. to be honest I did not verify that removing this would make a regression wrt error messages...